.github/workflows/skywalking.yaml

```diff
@@ -662,6 +662,8 @@ jobs:
             config: test/e2e-v2/cases/virtual-mq/e2e.yaml
           - name: AWS Cloud EKS
             config: test/e2e-v2/cases/aws/eks/e2e.yaml
+          - name: WINDOWS
```

Comment:
```suggestion
          - name: Windows
```

Reply:
The tests are still failing, it is better you could verify locally rather than costing a lot of upstream CI resources.

Reply:
Ok, this is my first time writing e2e of skywalking,  I will follow your suggestion and run successfully locally first.